src/Webnative.elm

```diff
@@ -109,10 +110,23 @@ type alias AppPermissions =
 
 
 {-| Filesystem permissions.
+
+    ```elm
+    import Webnative.Path as Path
+
+    { private =
+        [ Path.forPermissions (Path.directory [ "Audio", "Mixtapes" ])
+        , Path.forPermissions (Path.directory [ "Audio", "Playlists" ])
+        ]
+    , public =
+        []
+    }
```

Comment:
I see that using `Path t` is not an option here.

What do you think of
```elm
{ private =
  { files = [ ... ]
  , directories = [ ... ]
  }
, ...
}
```
?
Or maybe `privateFiles` and `privateDirectories`.

I don't think there's that much we gain by being as similar as possible to the typescript version. In the end I think ideally they diverge anyway, as the ts version tries to be nicely usable for typescript and the elm version tries to fit as good as possible into elm.

Reply:
Oh ya, `private.files` is much better, I'll do that 👍